pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -292,33 +292,50 @@ func TestGetPipelineFunc_RemoteResolution(t *testing.T) {
 	ctx = config.ToContext(ctx, cfg)
 	pipeline := parse.MustParseV1beta1Pipeline(t, pipelineYAMLString)
 	pipelineRef := &v1beta1.PipelineRef{ResolverRef: v1beta1.ResolverRef{Resolver: "git"}}
-	pipelineYAML := strings.Join([]string{
-		"kind: Pipeline",
-		"apiVersion: tekton.dev/v1beta1",
-		pipelineYAMLString,
-	}, "\n")
 
-	resolved := test.NewResolvedResource([]byte(pipelineYAML), nil, sampleConfigSource.DeepCopy(), nil)
-	requester := test.NewRequester(resolved, nil)
-	fn := resources.GetPipelineFunc(ctx, nil, nil, requester, &v1beta1.PipelineRun{
-		ObjectMeta: metav1.ObjectMeta{Namespace: "default"},
-		Spec: v1beta1.PipelineRunSpec{
-			PipelineRef:        pipelineRef,
-			ServiceAccountName: "default",
-		},
-	})
+	testcases := []struct {
+		name         string
+		pipelineYAML string
+	}{{
+		name: "v1beta1 pipeline",
+		pipelineYAML: strings.Join([]string{
+			"kind: Pipeline",
+			"apiVersion: tekton.dev/v1beta1",
+			pipelineYAMLString,
+		}, "\n"),
+	}, {
+		name: "v1 pipeline",
+		pipelineYAML: strings.Join([]string{
+			"kind: Pipeline",
+			"apiVersion: tekton.dev/v1",
+			pipelineYAMLString,
+		}, "\n"),
+	}}
+	for _, tc := range testcases {
+		t.Run(tc.name, func(t *testing.T) {
+			resolved := test.NewResolvedResource([]byte(tc.pipelineYAML), nil, sampleConfigSource.DeepCopy(), nil)
```

Comment:
nit: can you include inline comments explaining what the nil values are? e.g. `test.NewResolvedResource([]byte(tc.pipelineYAML), nil /* my-special-input */, ...)` (and same for the taskref tests)

Reply:
Added